api-references/supervisely-annotation-json-format/point-cloud-episodes.md

```diff
@@ -4,63 +4,86 @@
 
 ## Project Structure Example
 
-Root folder for the project named `project name`
-
-* `meta.json` file
-* `key_id_map.json` file
-* Dataset folders, that represents single episode. Each named `dataset_name`, which contains:
-  * `annotation.json` - file with whole episode annotation
-  * `frame_pointcloud_map.json` - file with pointcloud to episode frame mapping
-  * `pointcloud` folder, contains source point cloud files, for example `frame1.pcd, frame2.pcd`
-  * `related_images` optional folder, contains photo-context data:
-    * Frame folder, each named according to pointcloud (`/related_images/frame1/`), which contains:
-      * image files (`.png \ .jpg`)
-      * image ann files (`.json`) - json files, named according to image name (`1.png -> 1.json`)
+```text
+<PROJECT_NAME>
+├── key_id_map.json (optional)                
+├── meta.json     
+├── <EPISODE_NAME_1>                       
+│ ├── annotation.json           
+│ ├── frame_pointcloud_map.json     
+│ ├── pointcloud                    
+│ │   ├── scene_1.pcd           
+│ │   ├── scene_2.pcd   
+│ │   └── ...                
+│ └── related_images (optional)        
+│     ├── scene_1_pcd               
+│     │ ├── scene_1_cam0.png       
+│     │ ├── scene_1_cam0.png.json  
+│     │ ├── scene_1_cam1.png       
+│     │ ├── scene_1_cam1.png.json  
+│     │ └── ... 
+│     ├── scene_2_pcd               
+│     │ ├── scene_2_cam0.png       
+│     │ ├── scene_2_cam0.png.json  
+│     │ ├── scene_2_cam1.png       
+│     │ ├── scene_2_cam1.png.json  
+│     │ └── ... 
+│     └── ...      
+├── <EPISODE_NAME_2>                       
+│ ├── annotation.json               
+│ ├── frame_pointcloud_map.json     
+│ ├── pointcloud                    
+│ │   ├── scene_1.pcd                 
+│ │   └── ...               
+│ └── related_images (optional)          
+│     ├── scene_1_pcd               
+│     │ ├── scene_1_cam0.png       
+│     │ ├── scene_1_cam0.png.json    
+│     │ └── ... 
+│     ├── scene_2_pcd               
+│     │ ├── scene_2_cam0.png       
+│     │ ├── scene_2_cam0.png.json  
+│     │ └── ... 
+│     └── ...                    
+└── <EPISODE_NAME_...>                       
+```
 
 ## Main concepts
 
-**Point cloud Episode Project Folder**
+**Point cloud Episodes Project**
 
-On the top level are stored Project folders, these are the elements visible on the main Supervisely dashboard. Inside them they can contain only Datasets and Poject Meta information, all other data has to be stored a level below in a Dataset (Sequence). All datasets within a project have to contain content of the same cathegory.
+Point cloud Episode Project is format to support sequential point cloud data. Project consists from episodes - sequenital point cloud scenes. The first level inside the project folder contains folders with episodes, file with project meta (meta.json) and file with correspondences between project entities identifiers from Supervisely and local keys (key_id_map.json).
```

Comment:
Point cloud Episodes (PCE) Project consists of one or many sequences of frames.
Each sequence is called an episode/dataset in Supervisely.

PCE also includes Sensor fusion feature that supports video camera sensor in the Labeling Tool UI